Purchases/Networking/HTTPClient.swift

```diff
@@ -169,19 +167,19 @@ private extension HTTPClient {
                 urlRequest: URLRequest,
                 data: Data?,
                 error networkError: Error?) {
-        var statusCode = HTTPStatusCode.networkConnectTimeoutError.rawValue
+        var statusCode: HTTPStatusCode = .networkConnectTimeoutError
         var jsonObject: [String: Any]?
         var httpResponse: HTTPResponse? = HTTPResponse(statusCode: statusCode, jsonObject: jsonObject)
         var receivedJSONError: Error?
 
         if networkError == nil {
             if let httpURLResponse = urlResponse as? HTTPURLResponse {
-                statusCode = httpURLResponse.statusCode
+                statusCode = .init(rawValue: httpURLResponse.statusCode)
```

Comment:
maybe it's worth making `HTTPStatusCode` conform to `ExpressibleByIntegerLiteral` so we can just directly assign?

Reply:
Oh yeah let me try that!

Reply:
I added that conformance, but it doesn't work for assignments like this.